opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -13,13 +13,22 @@
 # limitations under the License.
 
 import logging
-from typing import Sequence, Tuple, Type
+from collections import OrderedDict
+from typing import Dict, Sequence, Tuple, Type
 
 from opentelemetry import metrics as metrics_api
 
 logger = logging.getLogger(__name__)
 
 
+class LabelSet(metrics_api.LabelSet):
+    """See `opentelemetry.metrics.LabelSet."""
+
+    def __init__(self, labels: Dict[str, str] = None, encoded: str = ""):
+        self.labels = labels
```

Comment:
Is it required to use a particular LabelSet implementation with a specific Meter implementation? I notice that there's no methods or properties for the API, but this is exposing the "labels" and "encoded" properties which are being utilized.

Reply:
That is a good question. Yes, a particular LabelSet implementation should only work with a specific Meter implementation. I've included validation in the metric methods for this. If they do not match, we return the empty label set.